pkg/apis/config/defaults.go

```diff
@@ -59,7 +59,7 @@ const (
 	DefaultAllowContainerConcurrencyZero bool = true
 )
 
-func defaultConfig() *Defaults {
+func defaultDefaultsConfig() *Defaults {
```

Comment:
`defaultServingConfig` ?

Reply:
But the config is really named `Defaults` :smile: